apps/files/templates/appnavigation.php

```diff
@@ -43,7 +43,7 @@
 			<label for="webdavurl"><?php p($l->t('WebDAV')); ?></label>
 			<input id="webdavurl" type="text" readonly="readonly"
 				   value="<?php p($_['webdav_url']); ?>"/>
-			<em><?php print_unescaped($l->t('Use this address to <a href="%s" target="_blank" rel="noreferrer noopener">access your Files via WebDAV</a>', array(link_to_docs('user-webdav')))); ?></em>
+			<em><?php print_unescaped($l->t('<a href="%s" target="_blank" rel="noreferrer noopener">Use this address to access your Files via WebDAV</a>', array(link_to_docs('user-webdav')))); ?></em>
```

Comment:
As long as we are fixing things. Could you move out the `a` parts of the translation?

Reply:
@rullzer Do you mean something like this?
```suggestion
			<em><?php print_unescaped(<a href="%s" target="_blank" rel="noreferrer noopener">, $l->t('Use this address to access your Files via WebDAV'), </a>, array(link_to_docs('user-webdav'))); ?></em>
```

Reply:
Yes. No need to have our translaters translate the link stuff :)

Reply:
okay, this is done 👍